server/item/register.go

```diff
@@ -145,6 +145,10 @@ func init() {
 		world.RegisterItem(SplashPotion{Type: p})
 		world.RegisterItem(Potion{Type: p})
 	}
+	world.RegisterItem(Template{TemplateNetheriteUpgrade()})
+	for _, t := range Templates() {
+		world.RegisterItem(Template{Template: t})
+	}
```

Comment:
Why is TemplateNetheriteUpgrade() separate?

Reply:
Because there’s no reason users should be able to access it since the template just converts it to nether rite and if they do add the template this way it just feels redundant since they are able to just give them netherrite armor in the first place.